reference/twig_reference.rst

```diff
@@ -636,6 +636,37 @@ serialize
 Accepts any data that can be serialized by the :doc:`Serializer component </serializer>`
 and returns a serialized string in the specified ``format``.
 
+.. _reference-twig-filter-emojify:
+
+emojify
+~~~~~~~
+
+.. versionadded:: 7.1
+
+    The ``emojify`` filter was introduced in Symfony 7.1.
+
+.. code-block:: twig
+
+    {{ text|emojify(catalog = null) }}
+
+``text``
+    **type**: ``string``
+
+``catalog`` *(optional)*
```

Comment:
When is null, default extension catalog is applied. It's "text" by default in construct.

I don't remember what is text behavior, but we should discribe this default ?

Reply:
`text` locale must be documented first https://github.com/symfony/symfony-docs/issues/19745

Reply:
I created https://github.com/symfony/symfony-docs/pull/19919